references/settings.md

```diff
@@ -44,11 +44,11 @@ List the settings.
 }
 ```
 
-## Add or update settings
+## Add or replace settings of an index
```

Comment:
```suggestion
## Add or replace the settings of an index
```

This is not a correct English sentence however, but I understand its too long for the sidebar.